### ISSUE TITLE
Firewall issue

### ISSUE THREAD

**Issue by Geetarman** [Azure/azure-documentdb-dotnet#431]
In the firewall settings for my cosmos db account I have the setting "Allow Access to Azure portal" turned OFF.
I can't see what this is preventing me from doing in the portal. I can still browse and view documents, I can still scale the settings up and change the indexing policy etc.

Is this not working correctly or is it a misunderstanding on my part.

regards

**Comment by hemantkr** [comment]
Can you please confirm that you haven't enabled Allow access to azure services. If you have enabled this setting it allows any service deployed in Azure to access the database. Therefore, the portal IPs are also allowed.

**Comment by Geetarman** [comment]
OK - this is confusing as the portal doesn't really indicate this is the case.  With 'Allow access to Azure Portal' OFF and 'Allow Access to Azure Services' ON it is displaying the following message.

Without portal access enabled, you will not be able to perform any operations on collections or documents within this account from the portal. Please turn on “Allow access to Azure Portal” if portal access is needed.

Basically I want to be able to deploy our web app but not allow DevOps access to the data...
